src/glass/src/app/pages/dashboard-page/dashboard-page.component.ts

```diff
@@ -39,6 +39,10 @@ export class DashboardPageComponent implements OnInit {
     {
       id: 'hosts',
       title: TEXT('Hosts')
+    },
+    {
+      id: 'services-capacity',
```

Comment:
I'm wondering if it should be shown by default. So... I mean to add `services-capacity` to `enabled`:

```
enabled: string[] = ['health', 'capacity', 'services', 'hosts', 'services-capacity'];
```

Reply:
Sounds like a good idea - I'll fix that afterwards.